testsuite/testsuite-opentracing/src/test/java/org/wildfly/swarm/opentracing/MockTracerResolver.java

```diff
@@ -3,13 +3,13 @@
 import io.opentracing.Tracer;
 import io.opentracing.contrib.tracerresolver.TracerResolver;
 import io.opentracing.mock.MockTracer;
-import io.opentracing.util.ThreadLocalActiveSpanSource;
+import io.opentracing.util.ThreadLocalScopeManager;
 
 /**
  * @author Juraci Paixão Kröhling
  */
 public class MockTracerResolver extends TracerResolver {
-    static final MockTracer TRACER_INSTANCE = new MockTracer(new ThreadLocalActiveSpanSource(), MockTracer.Propagator.TEXT_MAP);
+    static final MockTracer TRACER_INSTANCE = new MockTracer(new ThreadLocalScopeManager(), MockTracer.Propagator.TEXT_MAP);
```

Comment:
Can use the default constructor now, which uses these same parameters.

Reply:
Thanks!